### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by nctl144**
This PR updates the Readme for the project with more information on the performance profiling result!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +48,92 @@ make build_ext
...
+
+`make install`
+
+This will run `python setup.py install` command on this project.
+
+`make develop`
+
+This will run `python setup.py develop` command on this project.
+
+`perf`
```
Comment (lopuhin) [inline README.md]: Should be `make perf`?

File: README.md [hunk 2]
```diff
@@ -49,12 +48,92 @@ make build_ext
...
+
+Run the performance tests on `urlparse`, `urlsplit` and `urljoin`.
+
+cano:
```
Comment (lopuhin) [inline README.md]: and ``make cano``, with quotes

File: README.md [hunk 3]
```diff
@@ -49,12 +48,92 @@ make build_ext
...
+```
+
+Then you can run `python benchmarks/cython_profile.py --func [function-name]` to get the
+cprofiling result. Currently, Scurl supports profiling `urlparse`, `urlsplit` and `canonicalize`.
+
+This is not the most convenient way to profile Scurl with cprofiler, but we will come
+up with a way of improving this soon!
+
+
+## Profiling result report
```
Comment (lopuhin) [inline README.md]: I would say this is benchmarking, not profiling

File: README.md (hunk 1 above)
Comment (nctl144) [reply README.md]: you are right! I will change all of these based on your suggestions 😄

**Pull request closed without merging**